test/carotation/workloads/sleep.yaml

```diff
@@ -61,7 +61,7 @@ spec:
       serviceAccountName: sleep
       containers:
       - name: sleep
-        image: quay.io/joshvanl_jetstack/curl
+        image: quay.io/curl/curl:latest
```

Comment:
note: These curl images on quay.io are official, see https://github.com/curl/curl-container

I was suspicious of them initially because we so rarely see images on quay.io except our own!